### PR TITLE
Trim padding

### PR DESCRIPTION
**Pull request by gsganden**
I would like to remove white padding from some images as I download them.

Pull Request Checklist
 - [x] Pull request is either from WIP to master or from some other branch to WIP.
 - [x] Pull request includes a description of the change and the reason behind it.
 - [x] Pull request includes unit tests for any new functionality. 
 - [x] `./.ci/test.sh` passes locally.
 - [x] `README.md` and other docs have been updated as needed.

Maintainer's responsibilities 
 - [x] `CHANGELOG.md` has been updated.
 - [x] `_version.py` has been updated.

### CODE REVIEW

File: creevey/ops/image.py [hunk 1]
```diff
@@ -143,3 +143,78 @@ def centercrop(image: np.array, reduction_factor: float, **kwargs):
...
+) -> np.array:
+    """
+    Remove padding from an image
+
+    Remove rows and columns on the edges of the input image where the
+    brightness on a scale of 0 to 1 satisfies `comparison_op` with
+    respect to `threshold`. Brightness is evaluated by converting to
+    grayscale and normalizing if necessary. For instance, using
+    `threshold=.95` and `comparison_op=operator.gt` will result in
+    removing near-white padding, while using using `threshold=.05` and
```
Comment (htorrence) [inline creevey/ops/image.py]: Looks like `threshold` changed to `thresh`

File: creevey/ops/image.py [hunk 2]
```diff
@@ -143,3 +143,78 @@ def centercrop(image: np.array, reduction_factor: float, **kwargs):
...
+        return image
+
+
+def _convert_to_grayscale(image: np.array) -> np.array:
+    """
+    Convert image to grayscale.
+
+    Assumes image is grayscale, RGB, or RGBA.
+    """
+    grayscale = image.ndim == 2
```
Comment (htorrence) [inline creevey/ops/image.py]: [optional] I find the assignment/ equality check precedence hard to parse when reading through code quickly, how do you feel about adding parentheses here to make it obvious? 
```suggestion
    grayscale = (image.ndim == 2)
```

File: tests/test_ops/test_ops_image.py [hunk 3]
```diff
@@ -133,3 +135,70 @@ def test_centercrop_accepts_custom_reporting_args(sample_image_square_rgb):
...
+    np.testing.assert_almost_equal(actual, sample_image_tall_grayscale)
+
+
+def test_trim_black_padding_rgb(sample_image_square_rgb):
+    im_padded = _add_black_padding(sample_image_square_rgb)
+    actual = trim_padding(im_padded, thresh=0.05, comparison_op=operator.lt)
+    np.testing.assert_almost_equal(actual, sample_image_square_rgb)
+
+
+def _add_black_padding(im):
```
Comment (htorrence) [inline tests/test_ops/test_ops_image.py]: Why define this additional function instead of using `cv.copyMakeBorder` for the last test as well?

File: creevey/ops/image.py (hunk 1 above)
Comment (gsganden) [reply creevey/ops/image.py]: Fixed

File: tests/test_ops/test_ops_image.py (hunk 3 above)
Comment (gsganden) [reply tests/test_ops/test_ops_image.py]: Oh, right. I was thinking that `cv.copyMakeBorder` could only make a black border, but that's not true.

File: creevey/ops/image.py (hunk 2 above)
Comment (gsganden) [reply creevey/ops/image.py]: I agree, but `black` takes the parens out, and I'd rather use `black` and not override it here than satisfy this preference.

File: tests/test_ops/test_ops_image.py (hunk 3 above)
Comment (gsganden) [reply tests/test_ops/test_ops_image.py]: Fixed
